src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -57,6 +58,14 @@ public IPackageInstaller PackageInstaller
             get { return dynamoViewModel.PackageManagerClientViewModel; }
         }
 
+        /// <summary>
+        /// A reference to list of Local loaded packages
+        /// </summary>
+        public System.Collections.Generic.IEnumerable<Package> LocalPackages
```

Comment:
I find this super weird - Can you avoid adding anything here for now and lets just get the ExtensionsManager and then the PackageManagerExtension to avoid polluting this class until we have a plan?